CHANGELOG

```diff
@@ -85,6 +85,8 @@ Core:
 - Added custom translations in config [#3383].
 - Added dynamic webpage title [#3404].
 - Added 'go to top'-link [#3404].
+- Added caching for the index views. When using a Webserver for serving
```

Comment:
Please add a handy example configuration to a file like `DEVELOPMENT.rst` or something similar and remove this hint here.

Reply:
Done.